sasl/aws_msk_iam/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/segmentio/kafka-go/sasl/aws_msk_iam
+
+go 1.15
+
+require (
+	github.com/aws/aws-sdk-go v1.41.3
+	github.com/segmentio/kafka-go v0.4.23
```

Comment:
I think this still needs to refer to a kafka-go version after the sasl changes have been merged? I also have no idea how the replace statement will work if this is imported somewhere else.

Reply:
Ah yeah, good catch. I'll remove that replace statement, then update the version here. I'm not 100% sure how we'll make sure they stay in sync, but we'll cross that bridge after merge.